docker-compose/database.properties

```diff
@@ -3,4 +3,5 @@ host=172.17.0.1
 login=postgres
 password=postgres
 port=5432
-
+driverClassName=org.postgresql.Driver
+url=jdbc:postgresql://172.17.0.1:5432
```

Comment:
redundant infos, i think we should update actions-server  to use  driver instead of driverClassName 
and we should remove url and use host and port instead.
also we should use login instead of username in the application.properties file !

Reply:
Done